Sources/SourceKitLSP/TestDiscovery.swift

```diff
@@ -15,6 +15,11 @@ import LSPLogging
 import LanguageServerProtocol
 import SwiftSyntax
 
+public enum TestStyle {
```

Comment:
We call this `Library` in SwiftPM, FWIW. Consistency may be a good thing.

Reply:
I was hoping that we could maybe upstream this request to the LSP spec, at which point it would become language-agnostic. And at that point, I think it’s better to have a general term like *Style* instead of *Library*, which has more implications about build/etc structure.

Reply:
I don't feel strongly about either term. :)